code/modules/mob/living/carbon/xenomorph/castes/slime/slime.dm

```diff
@@ -0,0 +1,36 @@
+/mob/living/carbon/xenomorph/slime
+	caste_base_type = /mob/living/carbon/xenomorph/slime
+	name = "Slime"
+	desc = "A viscous, squishy and oozy substance. It quivers every now and then, as if it were alive."
+	icon = 'icons/Xeno/1x1_Xenos.dmi'
+	icon_state = "Slime Walking"
+	speak_emote = list("blurbs", "blorbs")
+	attacktext = "burns"
+	bubble_icon = "alienleft"
+	alpha = 200
+	health = 225
+	maxHealth = 225
+	plasma_stored = 125
+	pixel_x = 0
+	old_x = 0
+	pull_speed = -2
+	mob_size = MOB_SIZE_HUMAN
+	tier = XENO_TIER_ONE
+	upgrade = XENO_UPGRADE_ZERO
+	flags_pass = PASSTABLE
```

Comment:
```suggestion
	flags_pass = PASSTABLE | PASSMOB | PASSXENO
```

Would be funny if it can go through Xenos and marines and not shuffle them. (Semi realistic too)

Or on second thought that would be a neat mobility ability.